src/octoprint/printer/standard.py

```diff
@@ -369,10 +369,14 @@ def extrude(self, amount, speed=None, *args, **kwargs):
 
 		# Use specified speed (if any)
 		extrusion_speed = speed
+		max_e_speed = printer_profile["axes"]["e"]["speed"]
```

Comment:
IMHO `max...` is a misnomer since at this point it isn't the result of some calculation, is is just read from the profile.
I'd rather use something like `configured_e_speed`.

Reply:
It is correct that it is the configured value in the admin console. The UI says "Please define the maximum speed/feedrate of the individual axes and whether their control should be inverted or not.". That means that the configured value is also the maximum value for that printer profile. I like being explicit about the semantics/meaning of the variables. If you guys still prefer to be named configured and drop max then I can do the change.

Reply:
Considering that it's based on the semantics in the printer profile configuration, I think that the variable name is fine here. It raises the point though if we have a wording issue here in the printer profile dialog, or maybe in the usage of those speed settings overall. 

They are used for a) the feedrate for jogging/extruding and b) the initial feedrate in the gocde analysis step if no feedrate has been defined yet. And now I'm wondering how best to call that in the UI ;) Probably more of a default speed considering how it's used than an actual max speed. In any case, something that can wait until 1.3.11.